coffee_roaster_market/apps/coffee/tests/test_views.py

```diff
@@ -0,0 +1,252 @@
+import pytest
+import json
+
+from typing import Final, Optional, Iterable
+from django.contrib.auth.models import User
+from rest_framework.test import APIClient
+
+from coffee_roaster_market.apps.coffee.serializers import (
+    SensorialProfilSerializer,
+    GeoLocationSerializer,
+    PlantationSerializer,
+    CoffeeSerializer,
+)
+
+from .factories import (
+    SensorialProfileFactory,
+    GeolocationFactory,
+    PlantationFactory,
+    CoffeeFactory,
+)
+from .assets import parse_json_data
+
+USERNAME: Final[str] = "someuser"
+PASSWORD: Final[str] = "password"
+JSON_FORMAT: Final[str] = "json"
+
+SENSORIAL_PROFILES_URL: Final[str] = "/sensorial_profiles/"
+GEOLOCATIONS_URL: Final[str] = "/geolocations/"
+PLANTATIONS_URL: Final[str] = "/plantations/"
+COFFEE_URL: Final[str] = "/coffee/"
+
+pytestmark = pytest.mark.django_db
+
+
+def compare_dicts(
+    value: dict, expected_value: dict, exclude_fields: Optional[Iterable[str]] = None
+) -> None:
+    exclude_fields = exclude_fields or []
+    keys = (key for key in expected_value if key not in exclude_fields)
+    for key in keys:
+        assert value[key] == expected_value[key]
+
+
+@pytest.fixture
+def client(django_user_model: User) -> APIClient:
+    user = django_user_model.objects.create_user(username=USERNAME, password=PASSWORD)
+    client = APIClient()
+    client.force_authenticate(user=user)
+
+    return client
+
+
+def test_get_sensorial_profile(client: APIClient):
+    sensorial_profile = SensorialProfileFactory()
+    serialized = SensorialProfilSerializer(sensorial_profile)
+    response = client.get(SENSORIAL_PROFILES_URL)
+
+    assert response.status_code == 200
```

Comment:
Please check `from rest_framefork import status`

Reply:
Great idea, it's done